source/modules/MEI.xml

```diff
@@ -478,9 +478,7 @@
     </content>
   </macroSpec>
   <macroSpec ident="data.CERTAINTY" module="MEI" type="dt">
-    <desc>Values for certainty attribute. Certainty may be expressed by one of the values 'high',
-      'medium', or 'low'. The value 'unknown' should be used in cases where the encoder does not
-      wish to assert an opinion.</desc>
+    <desc xml:lang="en">Values for certainty attribute.</desc>
```

Comment:
Why did the further description go to remarks?

Reply:
This one was copied over directly from TEI, see https://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-teidata.certainty.html 
Maybe it would be good to also copy "defines the range of attribute values expressing a degree of certainty."?

Reply:
It took me a while to understand that the description in TEI is only in the remarks, so you copied it over here from desc to remarks to follow the original modelling in TEI, right? If so, would that need to be checked with other datatypes, attributes, etc., as well, and how is it connected to the title and description of this PR here? (Sorry to be annoying about the latter, but it makes things easier to review and follow if there are less "side effects" not connected to the actual PR.)

Reply:
It felt easier to just copy the whole block, but if this awkward, I'll revert this.

Reply:
Yes, please. I think it is better to have a separate PR for this change, where the content of desc could also be updated according to your proposal.